ios/app/MBXViewController.mm

```diff
@@ -186,7 +186,7 @@ - (void)parseFeaturesAddingCount:(NSUInteger)featuresCount
 
         if ([features isKindOfClass:[NSDictionary class]])
         {
-            NSMutableArray *annotations = [NSMutableArray array];
+            NSMutableArray <MBXAnnotation *> *annotations = [NSMutableArray array];
```

Comment:
Heh, sorry @incanus, that was in response to @friedbunny. (I was using GitHub’s mobile version, which doesn’t poll for updates.)